src/routes/write-exercise/components/AnswerSection.tsx

```diff
@@ -200,7 +200,7 @@ const AnswerSection = ({
   }
 
   return (
-    <>
+    <Pressable onPress={() => Keyboard.dismiss()}>
```

Comment:
Do we need it after all?

Reply:
yes when clicking not on the image but on the background.